### PR TITLE
15.0 mig mgmtsystem review

### PR DESCRIPTION
**Pull request by YvanDotet**

### CODE REVIEW

File: mgmtsystem_review/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "data": [
+        "security/ir.model.access.csv",
+        "security/mgmtsystem_review_security.xml",
+        "data/ir_sequence.xml",
+        "views/mgmtsystem_review.xml",
+        "views/res_users.xml",
+        "report/review.xml",
+        "report/report.xml",
+    ],
+    "development_status": "Production/Stable",
```
Comment (ivantodorovich) [inline mgmtsystem_review/__manifest__.py]:
```
Run manifestoo -d . check-dev-status --default-dev-status=Beta
mgmtsystem_review (Production/Stable) depends on mgmtsystem_nonconformity (Beta)
mgmtsystem_review (Production/Stable) depends on mgmtsystem_survey (Beta)
Error: Process completed with exit code 1.
```

```suggestion
```

Either remove this, or bump all the dependency chain to `"Production/Stable"` (including dependencies in other repos)

**Review by ivantodorovich — APPROVED** [review]

File: mgmtsystem_review/__manifest__.py (hunk 1 above)
Comment (YvanDotet) [reply mgmtsystem_review/__manifest__.py]: @ivantodorovich 
Ok I will write " "installable": True, " ?
Thank you.

Comment (ivantodorovich) [reply mgmtsystem_review/__manifest__.py]: not needed but ok :)

Comment (YvanDotet) [reply mgmtsystem_review/__manifest__.py]: Ok and what is the meaning of 
    "development_status": "Production/Stable",
?
They are this on many modules on v13 and v14 now.
Thanks.

Comment (ivantodorovich) [reply mgmtsystem_review/__manifest__.py]: Here's the doc that describes it better than me: https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/oca_module_lifecycle_development_status.rst

Comment (YvanDotet) [reply mgmtsystem_review/__manifest__.py]: ok thank you

**Pull request merged**